code/modules/organs/subtypes/parasite.dm

```diff
@@ -215,17 +215,53 @@
 	icon_state = "blacktumor"
 	dead_icon = "blacktumor"
 
-	organ_tag = "zombie"
-
-	parent_organ = BP_CHEST
+	organ_tag = BP_ZOMBIE_PARASITE
+	parent_organ = BP_HEAD
 	stage_interval = 150
 
+	var/last_heal = 0
+	var/heal_rate = 5 SECONDS
+
 /obj/item/organ/internal/parasite/zombie/process()
 	..()
 
 	if (!owner)
 		return
 
+	if(length(owner.bad_external_organs) && last_heal + heal_rate < world.time && isundead(owner))
```

Comment:
This seems like a dangerous thing to run every 5 seconds. If zombies have regen, it should either be an active ability or zombies should be much frailer than they are right now, as they're pretty tanky.

Reply:
atm the time between regenerations increases by 2 seconds per regeneration, and it should be easy to trigger another break in the same place if one already happened. additionally, the head doesn't regenerate, which is the main target, which should make this less good. naturally, the time between regenerations can be increased by 5 seconds per, but i would like to playtest this setup first, ideally